src/iris/api.py

```diff
@@ -3684,11 +3693,14 @@ def get_api(config):
     debug = False
     if config['server'].get('disable_auth'):
         debug = True
-    master_sender = config['sender'].get('master_sender', config['sender'])
-    master_sender_addr = (master_sender['host'], master_sender['port'])
+
+    default_master_sender = config['sender'].get('master_sender', config['sender'])
```

Comment:
In this case, we're trying to get a master_sender subkey, but that might not exist, so we're defaulting to the parent sender key which should always exist. This can be made better in the future though.